noir-projects/noir-protocol-circuits/crates/types/src/abis/accumulated_data/private_accumulated_data_builder.nr

```diff
@@ -55,12 +55,12 @@ impl PrivateAccumulatedDataBuilder {
     pub fn to_combined(self, teardown_gas: Gas) -> CombinedAccumulatedData {
         // TODO(Miranda): Hash here or elsewhere?
```

Comment:
We now do a lot of log prep before converting to `CombinedAccumulatedData` - maybe it would be cleaner to have a separate `finalise_logs` fn in `common` to call here?